Debian/bin/lib/Cyrus/Docker/Command/build.pm

```diff
@@ -0,0 +1,48 @@
+use v5.36.0;
+
+package Cyrus::Docker::Command::build;
+use Cyrus::Docker -command;
+
+use Process::Status;
+
+sub execute ($self, $opt, $args) {
+  my $root = "/srv/cyrus-imapd";
+  chdir $root or die "can't chdir to $root: $!";
+
+  my $version = `./tools/git-version.sh`;
+  Process::Status->assert_ok("determining git version");
+
+  chomp $version;
+
+  say "building cyrusversion $version";
+
+  system('git clean -dfx');
+  Process::Status->assert_ok("git clean");
+
+  $ENV{CFLAGS}="-g -W -Wall -Wextra -Werror";
```

Comment:
Would be nice if this appended to CFLAGS rather than blatting it

Reply:
Let's fix this in a future update that adds more options.  That's going to be important to get right.